docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,144 @@
+
+Ensure each node that will be participating in the Tentacle Cluster is joined to the Active Directory Domain and has the **Failover Clustering** feature installed in Windows. For more information on installing the Failover Clustering feature in windows follow [this link](https://blogs.msdn.microsoft.com/clustering/2012/04/06/installing-the-failover-cluster-feature-and-tools-in-windows-server-2012/ "installing the failover cluster service feature and toold in windows server 2012")
+
+Open the **Failover Cluster Manager** console on one of the nodes. If there is no cluster configured yet, you can right click **Failover Cluster Manager** amd select **New Cluster**
+
+On the **Select Servers** page, enter the Fully Qualified Domain Name of each node that will be in this cluster.
+
+On the **Validation Warning** page, select `Yes, When I click Next, run configuration validation tests, and then return to the process of creating the cluster`
+
+The **Validate a Configuration Wizard** appears, select `Run all Tests` and select **Next**
+
+When all validation processes successfully, you will be returned to the **Create Cluster Wizard** where the **Access Point for Administering the Cluster** page appears. At this point, choose an IP Address that is on the same Network as both Tentacles and a hostname that is 15 characters or less. 
+
+*![](/docs/images/clustered-listening-tentacles/configure-clusterhostname.jpg)*
+
+:::warning
+**Access Point for Administering the Cluster**
+The IP Address which you specify here is not going to be used by Octopus Deploy Server, instead this is used for Administering the cluster.
+:::
+
+Now complete the wizard.
+
+## Adding Octopus Tentacle as a Generic Service Cluster {#ClusteringTentacles-AddTentacleCluster}
+
+Right-Click **Roles** and select **Configure Roles** then Highlight **Generic Service**, click **Next**.
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-servicetype.jpg)*
+
+Find and highlight **OctopusDeploy Tentacle** service in the list of available services, click **Next**
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-selectservice.jpg)*
+
+Under **Client Access Point** choose an apropriate NetBIOS Name and IP Address for this clustered role. Note down this IP Address/DNS Hostname, you will need it to add the Tentacle Cluster to your Octopus Server
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-clientaccess.jpg)*
+
+Under **Select Storage** choose the disk that is configured as shared storage.
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-storage.jpg)*
+
+Under **Replication Registry Settings** Add a new root registry key of "Software\Octopus" and complete the wizard.
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-key.jpg)*
+
+You have now successfully completed the installation of Octopus Deploy Tentacles in a new active/passive cluster, in listening mode.
```

Comment:
`You have now successfully completed the installation of Octopus Deploy Tentacles in a new active/passive cluster, in listening mode.` => `You have now successfully completed the installation of a new active/passive cluster with two listening Tentacles.`